scripts/esCleaner.py

```diff
@@ -0,0 +1,31 @@
+import elasticsearch
+import curator
+import sys
+
+
+def main():
```

Comment:
There should be a bash script that tells users how to use this, does pip install, etc and runs this script. The cron job can then be configured to run the script. Check out https://github.com/uber/jaeger/blob/master/plugin/storage/cassandra/schema/create.sh